pkg/operator/ceph/operator.go

```diff
@@ -120,13 +121,17 @@ func (o *Operator) Run() error {
 		}
 	}
 
+	// creating a context
+	stopContext, stopFunc := context.WithCancel(context.Background())
```

Comment:
please rename `stopFunc` to `contextCancel` it's clearer and `stopContext` just `ctx`, it's more common.

Reply:
also, how about using `defer` instead of calling `stopFunc()` everywhere?

Reply:
Good point... Yup made the change...